init-letsencrypt.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+read -p "Enter your domain name (example.com) : " DOMAIN_NAME
+domains=($DOMAIN_NAME evalai.$DOMAIN_NAME evalapi.$DOMAIN_NAME)
+
+touch .env
+echo "DOMAIN_NAME=$DOMAIN_NAME" > .env
+
+rsa_key_size=4096
+data_path="./certbot"
+
+# Adding a valid address is strongly recommended
+read -p "Enter your Email ID : " email
+
+staging=1 # Set to 1 if you're testing your setup to avoid hitting request limits
```

Comment:
why is this hardcoded to 1? can you take this a parameter to the script

Reply:
Yes, I can. The reason why I set it as 1 was that lets encrypt renews certs only a limited number of times (I think 7). So, for testing purpose we should use this